controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -441,11 +446,15 @@ def _prepare_mocks_for_get_snapshot_lsvolumesnapshot(self):
 
     def test_get_snapshot_lsvolumesnapshot_success(self):
         self._prepare_mocks_for_get_snapshot_lsvolumesnapshot()
-        snapshot = self.svc.get_snapshot("volume_id", "snapshot_name")
+        snapshot = self.svc.get_snapshot("volume_id", "snapshot_name", None, True)
         self.assertEqual("snapshot_name", snapshot.name)
         self.svc.client.svcinfo.lsvolumesnapshot.assert_called_once_with(filtervalue='snapshot_name=snapshot_name')
         self.svc.client.svcinfo.lsvdisk.assert_called_once_with(bytes=True, filtervalue='vdisk_UID=volume_id')
 
+    def test_get_snapshot_lsvolumesnapshot_not_supported_error(self):
+        with self.assertRaises(array_errors.Flashcopy2NotSupportedMessage):
+            self.svc.get_snapshot("volume_id", "snapshot_name", None, True)
```

Comment:
```suggestion
            self.svc.get_snapshot("volume_id", "snapshot_name", pool=None, flashcopy_2=True)
```

Reply:
why?

Reply:
the meaning of the hardcoded values is not clear without a name

Reply:
done

Reply:
LGTM